lib/perfdata/elasticsearchwriter.cpp

```diff
@@ -131,6 +132,36 @@ void ElasticsearchWriter::Pause()
 	ObjectImpl<ElasticsearchWriter>::Pause();
 }
 
+void ElasticsearchWriter::AddTemplateTags(const Dictionary::Ptr& fields, const Host::Ptr& host, const Service::Ptr& service, const CheckResult::Ptr& cr)
+{
+	MacroProcessor::ResolverList resolvers;
```

Comment:
You can declare this above or below the `ObjectLock olock ...` within the if block instead.